modules/gltf/gltf_state.cpp

```diff
@@ -358,3 +361,15 @@ String GLTFState::get_base_path() {
 void GLTFState::set_base_path(String p_base_path) {
 	base_path = p_base_path;
 }
+
+Variant GLTFState::get_additional_data(const String &p_extension_name) {
+	return additional_data[p_extension_name];
```

Comment:
Same here too.

Reply:
It returns null, which actually makes me realize, we don't need `has_additional_data`. I've documented this behavior and removed `has_additional_data`.